src/resources/elements/primeDesignSystem/demos/pselectDemo.html

```diff
@@ -1,7 +1,10 @@
 <template>
   <section class="demo">
     <h1>Simple Dropdown</h1>
-    <pselect data.bind="simpleData"></pselect>
+    --${test}--
+    <pselect data.bind="simpleData" value.bind="test"></pselect>
```

Comment:
Obsolete dev code?

Could we have a demo, where an array of `value`s is set?

Reply:
what is `  --${test}--` for?